haiku-apps/chat_o_matic/chat_o_matic-0.0.1.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A multi-protocol IM client"
+DESCRIPTION="Chat-O-Matic is a multi-protocol chat client. It supports \
+IRC, XMPP, and other protocols through libpurple."
+HOMEPAGE="https://github.com/JadedCtrl/Chat-O-Matic"
+COPYRIGHT="2009-2011 Andrea Anzani
+	2010-2015 Dario Casalinuovo
+	2009-2011 Pier Luigi Fiorini
+	2021 Jaidyn Levesque"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/JadedCtrl/Chat-O-Matic/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="bf61551b6d8cfc49373dd43a9fc0b2b27a566305be15a58a8817bea3522a33a3"
+SOURCE_DIR="Chat-O-Matic-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 !x86 x86_64"
+
+PROVIDES="
+	chat_o_matic = $portVersion
+	app:Chat_O_Matic = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libexpat
+	lib:libglib_2.0
+	lib:libgloox
+	lib:libpurple
+	lib:libssl
+	lib:libz
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libexpat
+	devel:libglib_2.0
+	devel:libgloox
+	devel:libpurple
+	devel:libssl
+	devel:libz
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:ld
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp objects/Chat-O-Matic $appsDir
+
+	mkdir -p $includeDir/chat-o-matic/
```

Comment:
Maybe they should go into a _devel subpackage?

Reply:
Just put headers into a devel package :)

Reply:
Any specific reason why the headers are included?

Reply:
It's just so add-ons can be made out-of-tree.